### ISSUE TITLE
PayPal account won't link to Game Jolt

### ISSUE THREAD

**Issue by noahmaue** [gamejolt/next-issue-tracker#1345]
**How are you using Game Jolt?**
As a freelance game developer.

**Please describe the issue you are having**
I'm trying to link a PayPal account to my Game Jolt account so I can pick the Pay What You Want option for my games. Each time I sign in with the PayPal account though, it says the page isn't working right now. Help?
![image](https://user-images.githubusercontent.com/32719997/69354510-3a1a3580-0c4e-11ea-830d-edf68afc9c0f.png)

**Comment by Caboaros** [comment]
I have the exact same problem. One and a half year trying already.

**Comment by Tricky1975** [comment]
It's raining complaints about 404 errors when people try to do anything PayPal related on Game Jolt. This this handles money, I think this is an issue that should get priority now.

**Comment by Redstoneinvente** [comment]
Same error here:

**Comment by Redstoneinvente** [comment]
Hi, I have fixed it. Try doing it on a smartphone

**Comment by Tricky1975** [comment]
Still it should work on a desktop/laptop as well

**Comment by Redstoneinvente** [comment]
Yeah, but idk where the problem is. Maybe there might be some plugins and
apis not installed properly maybe

Redstoneinvente
---------------------------
  Games Studio
